crates/holochain/src/core/workflow/integrate_dht_ops_workflow.rs

```diff
@@ -43,77 +58,168 @@ pub async fn integrate_dht_ops_workflow(
                     })?;
                 }
             }
-            let changed = txn
-                .prepare_cached(holochain_sqlite::sql::sql_cell::UPDATE_INTEGRATE_STORE_RECORD)?
-                .execute(named_params! {
-                    ":when_integrated": time,
-                    ":store_record": ChainOpType::StoreRecord,
-                })?;
+            add_stored_ops(
+                txn,
+                SELECT_STORE_RECORD_TO_INTEGRATE,
+                named_params! {
+                    ":dht_op_type": ChainOpType::StoreRecord,
+                },
+                &mut stored_ops,
+            )?;
+            let changed =
+                txn.prepare_cached(UPDATE_INTEGRATE_STORE_RECORD)?
+                    .execute(named_params! {
+                        ":when_integrated": time,
+                        ":store_record": ChainOpType::StoreRecord,
+                    })?;
```

Comment:
There's a lot up select-then-updates in this file now. Did the `RETURNING *` not work?

Reply:
A lot of? It's all select-then-update =D

Ha, I was wondering how to achieve that with a single query and didn't know about the `RETURNING` clause. Brilliant! I've tried it and it works perfectly. I'll refactor.

Reply:
done

Reply:
Woot!